corehq/apps/users/migrations/0001_add_location_permission.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-09 02:05
+from __future__ import unicode_literals
+from corehq.sql_db.operations import HqRunPython
+from corehq.apps.users.management.commands.add_location_permission import update_all_roles
+
+from django.db import migrations
+
+
+def migrate(apps, schema_editor):
+    update_all_roles()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        HqRunPython(migrate),
```

Comment:
The management command won't be run anywhere, actually.  I initially implemented it that way to be able to run it separately, but then realized it'd need to run as a migration.  Ideally I'd be able to run it from a setup_release before the deploy, then have the migration only run if the command hasn't been run manually already.  This would mean setting a flag somewhere in the command and checking for it in the migration.  That started to feel a little unwieldly, and there are only 93k relevant docs on prod, so I expect it to be quick enough not to matter.

Given that, this should probably all be in the migration at this point, but I also agree with Emord that this should noop after a while.  I think what I'd like to do is merge this as is, then open a PR to make it a noop, and merge that after a couple weeks.  Do y'all have thoughts on the best way to noop this?  Make the migration simply `return None`?  Use `CCHQ_IS_FRESH_INSTALL=1` (but that'd leave this code around forever)?  Can I delete the migration without messing up django's migration bookkeeping?